### PR TITLE
adding Checking keys in

### PR DESCRIPTION
**Pull request by julien51**
# Description

That's the blog post!

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: unlock-protocol.com/blog/checking-key-in.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Checking keys in
```
Comment (cnasc) [inline unlock-protocol.com/blog/checking-key-in.md]:
```suggestion
title: Checking Keys In
```

File: unlock-protocol.com/blog/checking-key-in.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+subTitle: Tickets to conferences are memberships!
+authorName: Julien Genestoux
+publishDate: February 6, 2020
+description: At Unlock, we're proud to power the tickets for several conferences... but no ticketing solution is complete without a checking mechanism!
```
Comment (cnasc) [inline unlock-protocol.com/blog/checking-key-in.md]:
```suggestion
description: At Unlock, we're proud to power the tickets for several conferences... but no ticketing solution is complete without a check-in mechanism!
```

File: unlock-protocol.com/blog/checking-key-in.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+image: /static/images/blog/introducing-tickets/introducing-tickets.jpg
+---
+
+Memberships are everywhere: from your local gym, to your favorite news site, through conferences! At Unlock, we take great care in making sure we can _bridge_ the online memberships and the offline world.
```
Comment (cnasc) [inline unlock-protocol.com/blog/checking-key-in.md]:
```suggestion
Memberships are everywhere: from your local gym, to your favorite news site, through conferences! At Unlock, we take great care in making sure we can _bridge_ the gap between online memberships and the offline world.
```

File: unlock-protocol.com/blog/checking-key-in.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+The verification page above works from any web browser... but, if viewed from inside a browser which has a crypto wallet and if the viewer is actually the lock owner, **the viewer can "mark" the ticket as checked-in**. For this, we are using metadata on the NFT which can only be set by the lock owner. This way, if the ticket is then transfered to someone else, the organizers would still be able to quickly identify sneaky users who are trying to re-use a ticket.
+
+
+<p style="text-align:center" ><img width="300" src="/static/images/blog/checking-key-in/checked-in-key.png" alt="checked-in!"></p>
+
+With our support for credit cards, the ability to ask users to submit information for their tickets, and sell them on the organizers' website directly, we believe Unlock now provides a **tickecting experience** that's even better than most legacy solutions, while still leveraging Ethereum's core capabilities: decentralization, permissionless and programmable!
```
Comment (cnasc) [inline unlock-protocol.com/blog/checking-key-in.md]:
```suggestion
With our support for credit cards, the ability to ask users to submit information for their tickets, and sell them on the organizers' website directly, we believe Unlock now provides a **ticketing experience** that's even better than most legacy solutions, while still leveraging Ethereum's core capabilities: decentralization, permissionless and programmable!
```

**Pull request merged**